test/util/triplea/mapXmlCreator/CanalDefinitionsPanelTest.java

```diff
@@ -7,11 +7,14 @@
 
   @Test
```

Comment:
@Ignore the test rather than commenting it out. At least we'll still get compile checking of the commented out code. Probably better is to just remove the test completely. @Ignore and commented out code are both not good things. In both cases it is dead code.